### PR TITLE
re #2 Remove dependency on `Globals`.

### PR DESCRIPTION
**Pull request by sallner**
Resolves the issue #2

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 import os
 
 
-version = '1.3.1'
+version = '2.0'
```
Comment (icemac) [inline setup.py]: I think this should be `2.0.dev0`.

File: CHANGES.rst [hunk 2]
```diff
@@ -2,11 +2,13 @@
 History
 =======
 
-1.3.1
+2.0
```
Comment (icemac) [inline CHANGES.rst]: To make sure `zest.releaser` finds this line it would be nice if it reads:
`2.0 (unreleased)`

**Review by icemac — APPROVED** [review]
LGTM besides some nitpicks.

**Comment by icemac** [comment]
@sallner Could you please rebase on `master` so your changes are run through TravisCI?

**Comment by sallner** [comment]
I adjusted the version numbers and will merged this PR now.

**Pull request merged**